### PR TITLE
Add the concept of time and change icons

### PR DESCRIPTION
**Pull request by ChloeEmmaA**
it is almost working, but the labels of the chart do not update correctly

### CODE REVIEW

**Comment by ChloeEmmaA** [comment]
To make the labels of the x axis, we worked  on the chart inclusion at different levels and included the chart.bundle from another chart project, this project is a nice example of a chart with time that change, I keep it in the file chart_time.html to take inspiration.
We still have to work to understand better why it works only with the chart.bundle from the other project and not the one we downloaded on the website.

**Comment by ChloeEmmaA** [comment]
:+1:

**Comment by ChloeEmmaA** [comment]
For now, the day change work only for the month of March 2018!

**Comment by Olivier0** [comment]
Test link: https://rawgit.com/ChloeEmmaA/open-sympto/add_calendar_and_update_variables/web/app.html

**Comment by ChloeEmmaA** [comment]
@Olivier0 , How I can get the link with the version that include the last commit?

**Comment by Olivier0** [comment]
The link I provided is correct. However, it usually takes a few minutes before a new commit is taken into account. Can you try again and confirm your last commit is used?

**Comment by ChloeEmmaA** [comment]
Yes, exactly, now it includes also the last commit! :)

**Comment by pletelli** [comment]
Did you draw the new icons ? They look great !

**Comment by pletelli** [comment]
I think the chart is not updated according to the observations filled in the form when I try with the test app. Should it be updated ?

**Comment by ChloeEmmaA** [comment]
@pletelli -
-the icons, yes, I draw them quickly in 1 hour in total, then that's the first draw, but I prefer to have something else than the sympto ones. I will still work on that.
-What do you mean exactly for the chart, as I said, it work only for the month of march and it should 1, centered in the middle the selected date, 2 display the temperature selected the selected day atnd the other selected temperature, 3. it choose now only for testing random values for the temperature not added (maybe the random function add confusion and we can already remove it), does that work for you or not? if not, what dosen't work exactly?

**Comment by ChloeEmmaA** [comment]
and you have to click always on "montrer les données choisies" to see the change (call the function that update the chart)

**Comment by Olivier0** [comment]
I would be better to have a "save" button instead. I guess "montrer les données choisies" was there for testing purposes.

**Comment by ChloeEmmaA** [comment]
Yes, it is for testing. I think maybe it is better to call the update function when the user click on "back" (instead of on a ok button), it decrease the number of clicks needed.

**Comment by Olivier0** [comment]
Yes, I guess you mean the "Close" button. Good idea. So, I would rename this "Close" button into "Save" then.

**Comment by Olivier0** [comment]
I believe we should delete the old images instead of moving them into an "old" directory. They will be removed from the current version, but still stored in github (in case we need them later on). That's cleaner and the usual way to proceed with git.

File: web/app/chart_time.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  <meta charset="utf-8">
+  <!--
+  Customize this policy to fit your own app's needs. For more guidance, see:
+      https://github.com/apache/cordova-plugin-whitelist/blob/master/README.md#content-security-policy
+  Some notes:
+      * gap: is required only on iOS (when using UIWebView) and is needed for JS->native communication
+      * https://ssl.gstatic.com is required only on Android and is needed for TalkBack to function properly
+      * Disables use of inline scripts in order to mitigate risk of XSS vulnerabilities. To change this:
+          * Enable inline JS: add 'unsafe-inline' to default-src
+        -->
```
Comment (Olivier0) [inline web/app/chart_time.html]: Can we remove this comment? It doesn't look very useful.

File: web/app/chart_time.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+<body class="color-theme-green">
+  
+  <script src="framework7/js/framework7.min.js"></script>
+  <script src="js/routes.js"></script>
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+<script type='text/javascript' src='https://blazingspider.com/wp-demo/wp-content/plugins/blazing-charts/js/chartjs/Chart.bundle.min.js?ver=1.0'></script>
+  <!--script src="js/lib/Chart.bundle.min.js"></script>
+  <!moment.js: date manipulations-->
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script
+  <script src="js/moment.min.js"></script>-->
```
Comment (Olivier0) [inline web/app/chart_time.html]: This commented out code should be removed.

File: web/app/chart_time.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  <script src="js/open-sympto.js"></script>
+
+
+
```
Comment (Olivier0) [inline web/app/chart_time.html]: Extra empty lines (not compliant with out linting rules).

File: web/app/index.html [hunk 4]
```diff
@@ -107,6 +107,11 @@
           </div>
         </div>
         <div class="page-content">
+
+
+
+
+
```
Comment (Olivier0) [inline web/app/index.html]: Empty lines (not compliant with our linting rules).

File: web/app/index.html [hunk 5]
```diff
@@ -576,8 +593,16 @@ <h3>Observations (pas d'impact sur la fenêtre de fertilité)</h3>
 -->
   <script src="framework7/js/framework7.min.js"></script>
   <script src="js/routes.js"></script>
-  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
-  <script src="js/lib/Chart.bundle.min.js"></script>
+  <script src="js/moment.min.js"></script>
+
+  <!--moment.js: date manipulations-->
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script-->
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/moment.min.js"></script-->
```
Comment (Olivier0) [inline web/app/index.html]: We should remove this commented out code.

File: web/app/index.html [hunk 6]
```diff
@@ -576,8 +593,16 @@ <h3>Observations (pas d'impact sur la fenêtre de fertilité)</h3>
...
+
+<script type='text/javascript' src='https://blazingspider.com/wp-demo/wp-content/plugins/blazing-charts/js/chartjs/Chart.bundle.min.js?ver=1.0'></script>
+  <!--script src="js/lib/Chart.bundle.min.js"></script-->
```
Comment (Olivier0) [inline web/app/index.html]: We should remove this commented out code.

File: web/app/chart_time.html (hunk 1 above)
Comment (ChloeEmmaA) [reply web/app/chart_time.html]: this file is only here for example (chart with dates that can be changed), we will completely remove this file afterward

File: web/app/chart_time.html (hunk 2 above)
Comment (ChloeEmmaA) [reply web/app/chart_time.html]: same, we will deletethis file

File: web/app/index.html (hunk 4 above)
Comment (ChloeEmmaA) [reply web/app/index.html]: ok

File: web/app/index.html (hunk 5 above)
Comment (ChloeEmmaA) [reply web/app/index.html]: done

File: web/app/index.html (hunk 6 above)
Comment (ChloeEmmaA) [reply web/app/index.html]: done

**Pull request merged**